tests/mocking/dummy.py

```diff
@@ -0,0 +1,138 @@
+from remoteappmanager.services.hub import Hub
+from remoteappmanager.services.reverse_proxy import ReverseProxy
+from remoteappmanager.file_config import FileConfig
+from remoteappmanager.application import Application
+from remoteappmanager.db import interfaces
+from remoteappmanager.docker.container_manager import ContainerManager
+from remoteappmanager.docker.container import Container
+from remoteappmanager.docker.image import Image
+
+from tests.utils import mock_coro_factory, basic_command_line_config
+
+
+class DummyDBApplication(interfaces.ABCApplication):
+    pass
+
+
+class DummyDBApplicationPolicy(interfaces.ABCApplicationPolicy):
+    pass
+
+
+class DummyDBAccounting(interfaces.ABCAccounting):
+
+    def get_user_by_name(self, user_name):
+        return user_name
+
+    def get_apps_for_user(self, account):
+        return (('12345',
+                 DummyDBApplication(image='image1'),
+                 DummyDBApplicationPolicy()),
+                ('id678',
+                 DummyDBApplication(image='image2'),
+                 DummyDBApplicationPolicy()))
+
+
+def create_reverse_proxy(params=None,
+                         register_result='',
```

Comment:
It would be convenient at this point to change mock_coro_factory to do exceptions as well as a return value, like mock does. It would simplify our general testing of exceptions.